docs/edit-contracts.md

```diff
@@ -0,0 +1,220 @@
+# How to edit zkEVM contracts for Kurtosis CDK
+
+A step-by-step guide to edit the zkEVM Solidity contracts in a Polygon CDK devnet.
+
+This document draws from the following resources:
+
+- [Polygon CDK Kurtosis Package](../README.md)
+- [Custom Docker Images for Kurtosis CDK](../docker/README.md)
+- [Video guide to the Polygon CDK](https://www.youtube.com/watch?v=6ykNLEhwxIs)
+
+## Goals
+
+- Edit the Solidity code in repo `zkevm-contracts` in file `VerifierRollupHelperMock.sol` in function `verifyProof`.
+- Rebuild the `zkevm-contracts` docker image.
+- Edit this repo `kurtosis-cdk` to use our new build of `zkevm-contracts`.
+- Spin up a Polygon CDK devnet and observe our changes in the active devnet.
+
+# Set up your system
+
+Follow the instructions at [Polygon CDK Kurtosis Package](../README.md) up to the terminal commands
```

Comment:
Add this note maybe as the version of kurtosis-cli I had installed was too recent:

```
Note that you should install kurtosis-cli=1.0.0. See https://docs.kurtosis.com/install-historical/.
```

Reply:
done in https://github.com/EspressoSystems/kurtosis-cdk/pull/2/commits/c2b262db4f7d1b6082c25063775e93aba88bb164